LICENSE

```diff
@@ -209,36 +209,39 @@ separate copyright notices and license terms:
 
 Library Google.Protobuf is under the following copyright:
 Copyright 2008 Google Inc. All rights reserved under BSD-3-Clause license
-(https://github.com/protocolbuffers/protobuf/blob/main/LICENSE).
+(<https://github.com/protocolbuffers/protobuf/blob/main/LICENSE>).
 
 Libraries
+
 - Google.Protobuf,
-- Grpc.Net.Common,
 - Grpc.Net.Client
+- Grpc.Net.Common,
 are under the following copyright:
 Copyright 2019 The gRPC Authors under Apache License Version 2.0
-(https://github.com/grpc/grpc-dotnet/blob/master/LICENSE).
+(<https://github.com/grpc/grpc-dotnet/blob/master/LICENSE>).
 
 Libraries
+
+- OpenTelemetry
 - OpenTelemetry.Api,
+- OpenTelemetry.Api.ProviderBuilderExtensions,
```

Comment:
There are only two non-trivial changes in this file. This is No. 1 of the two.

1. Renamed `OpenTelemetry.Extensions.DependencyInjection` to `OpenTelemetry.Api.ProviderBuilderExtensions`.